lib/cc/analyzer/engines_runner.rb

```diff
@@ -6,11 +6,12 @@ class EnginesRunner
       InvalidEngineName = Class.new(StandardError)
       NoEnabledEngines = Class.new(StandardError)
 
-      def initialize(registry, formatter, source_dir, config, container_label = nil)
+      def initialize(registry, formatter, source_dir, config, requested_paths = [], container_label = nil)
```

Comment:
Not a blocker, but this argument is getting long enough that keyword arguments are starting to look like a good option.

https://robots.thoughtbot.com/ruby-2-keyword-arguments